Automattic/wp-calypso
# Color update: Update blue and gray illustration colors

#### Changes proposed in this Pull Request

This updates all svg illustration images with the new brand colors from [Color Studio](https://automattic.github.io/color-studio/). This handles only blue and gray colors. I did not touch any Jetpack colors or any color that was not blue or gray.

If the old color was blue, I used the new `Blue` color. If the old color was gray, I used the new `Gray` color.

I basically just searched and replaced hex code colors. I did not open or save the files in any program, so the surrounding html should be unaffected.

Sorry for the large PR. It looks like the diff is not helpful for svg files that have been compressed.

#### Testing instructions

I've checked every image by quickly previewing the local svg file on my machine. Everything looks 👍 



* Check through the images and compared them to master to see if there are any bugs
* Check through Calypso and look at the illustrations

Explanation of the `+457 −476` diff:

Usually, it would be exactly +457 −457, but some of the illustrations used hex colors that were only slightly different in color. In those cases, they were consolidated into one.


Here are some example before/after:

<img width="1015" alt="screen shot 2019-01-18 at 4 18 55 pm" src="https://user-images.githubusercontent.com/618551/51415820-cb0b1680-1b3c-11e9-8ad7-9dbef01537a0.png">
<img width="1021" alt="screen shot 2019-01-18 at 4 18 30 pm" src="https://user-images.githubusercontent.com/618551/51415827-ce060700-1b3c-11e9-8a40-90c4d4af0e28.png">
<img width="1044" alt="screen shot 2019-01-18 at 4 19 38 pm" src="https://user-images.githubusercontent.com/618551/51415867-f42ba700-1b3c-11e9-82f7-f11564970cc7.png">
<img width="1043" alt="screen shot 2019-01-18 at 4 20 08 pm" src="https://user-images.githubusercontent.com/618551/51415872-f7bf2e00-1b3c-11e9-9fe8-a2e8eaa61c8d.png">
